Grace-Shopper-Dream-Team/Grace-Shopper
# logged in user cart functionality

# Context 
Needed to have a persistent cart for logged in users with CRUD abilities for their carts

# What this PR does
Displays a persistent cart for logged in user's even when they log out and come back in. Allows logged in users to perform CRUD operations on their carts. 

# How to test
Run server, open browser, login first user in database, and perform CRUD operations on cart. Log in and out to see if cart items persist.
